pipeline-scripts/openshiftv4_scale.groovy

```diff
@@ -0,0 +1,93 @@
+			def kubeconfig_file = scale_properties['KUBECONFIG_FILE']
+			def pbench_instrumentation = scale_properties['PBENCH_INSTRUMENTATION']
+			def enable_pbench_agents = scale_properties['ENABLE_PBENCH_AGENTS']
+			def enable_pbench_copy = scale_properties['ENABLE_PBENCH_COPY']
+			def pbench_ssh_private_key_file = scale_properties['PBENCH_SSH_PRIVATE_KEY_FILE']
+			def pbench_ssh_public_key_file = scale_properties['PBENCH_SSH_PUBLIC_KEY_FILE']
+			def pbench_server = scale_properties['PBENCH_SERVER']
+			def scale_ci_results_token = scale_properties['SCALE_CI_RESULTS_TOKEN']
+			def job_completion_poll_attempts = scale_properties['JOB_COMPLETION_POLL_ATTEMPTS']
+			def scale_test_prefix = scale_properties['SCALE_TEST_PREFIX']
+			def scale_metadata_prefix = scale_properties['SCALE_METADATA_PREFIX']
+			def scale_worker_count = scale_properties['SCALE_WORKER_COUNT']
+			def scale_poll_attempts = scale_properties['SCALE_POLL_ATTEMPTS']
+			def expected_scale_duration = scale_properties['EXPECTED_SCALE_DURATION']
+
+			try {
+				scale_build = build job: 'ATS-SCALE-CI-SCALE',
+				parameters: [   [$class: 'LabelParameterValue', name: 'node', label: node_label ],
+						[$class: 'StringParameterValue', name: 'CLUSTER_USER', value: cluster_user ],
+						[$class: 'StringParameterValue', name: 'CLUSTER_PASSWORD', value: cluster_password ],
+						[$class: 'StringParameterValue', name: 'CLUSTER_API_URL', value: cluster_api_url ],
+						[$class: 'StringParameterValue', name: 'SSHKEY_TOKEN', value: sshkey_token ],
+						[$class: 'StringParameterValue', name: 'ORCHESTRATION_HOST', value: orchestration_host ],
+						[$class: 'StringParameterValue', name: 'ORCHESTRATION_USER', value: orchestration_user ],
+						[$class: 'StringParameterValue', name: 'WORKLOAD_IMAGE', value: workload_image ],
+						[$class: 'BooleanParameterValue', name: 'WORKLOAD_JOB_NODE_SELECTOR', value: Boolean.valueOf(workload_job_node_selector) ],
+						[$class: 'BooleanParameterValue', name: 'WORKLOAD_JOB_TAINT', value: Boolean.valueOf(workload_job_taint)  ],
+						[$class: 'BooleanParameterValue', name: 'WORKLOAD_JOB_PRIVILEGED', value: Boolean.valueOf(workload_job_privileged)  ],
+						[$class: 'StringParameterValue', name: 'KUBECONFIG_FILE', value: kubeconfig_file ],
+						[$class: 'BooleanParameterValue', name: 'PBENCH_INSTRUMENTATION', value: Boolean.valueOf(pbench_instrumentation)  ],
+						[$class: 'BooleanParameterValue', name: 'ENABLE_PBENCH_AGENTS', value: Boolean.valueOf(enable_pbench_agents)  ],
+						[$class: 'BooleanParameterValue', name: 'ENABLE_PBENCH_COPY', value: Boolean.valueOf(enable_pbench_copy)  ],
+                                                [$class: 'StringParameterValue', name: 'PBENCH_SSH_PRIVATE_KEY_FILE', value: pbench_ssh_private_key_file ],
+                                                [$class: 'StringParameterValue', name: 'PBENCH_SSH_PUBLIC_KEY_FILE', value: pbench_ssh_public_key_file ],
```

Comment:
I think these two pbench options (`PBENCH_SSH_PRIVATE_KEY_FILE` and `PBENCH_SSH_PUBLIC_KEY_FILE`) can be removed since they get defined in the build step of the job.